src/app/components/ContentExcerpt.tsx

```diff
@@ -0,0 +1,59 @@
+import { Highlight } from '@openstax/highlighter/dist/api';
+import { HTMLAnchorElement } from '@openstax/types/lib.dom';
+import React from 'react';
+import { useSelector } from 'react-redux';
+import styled from 'styled-components/macro';
+import { content } from '../content/routes';
+import { book } from '../content/selectors';
+import { BookWithOSWebData } from '../content/types';
+import { fromRelativeUrl, getUrlParamForPageId } from '../content/utils/urlUtils';
+import { bodyCopyRegularStyle } from './Typography';
+
+// tslint:disable-next-line:variable-name
+export const HighlightContent = styled.div`
+  ${bodyCopyRegularStyle}
+  overflow: auto;
+
+  * {
+    overflow: initial;
+  }
+`;
+
+interface Props {
+  highlight: Highlight;
```

Comment:
content excerpts will not always use a highlight, there is an example signature in the card

Reply:
both HL and SG list element had `HighlightListElementProps {  highlight: Highlight; }` and they both do `data-highlight-id={highlight.id}`, is this different?

Reply:
content excerpts will not always use a highlight, you can accept a string for the excerpt